src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/add_driver_to_car")

```

Comment:
Fix your url pattern according to best practice
![image](https://user-images.githubusercontent.com/32339836/230099586-69e64d9b-ad22-4b7e-abc0-d2c4627cb085.png)